fleetshard/pkg/central/reconciler/reconciler_test.go

```diff
@@ -94,6 +125,35 @@ func TestReconcileCreate(t *testing.T) {
 	assert.Equal(t, centralReencryptRouteName, route.GetName())
 	assert.Equal(t, openshiftRouteV1.TLSTerminationReencrypt, route.Spec.TLS.Termination)
 	assert.Equal(t, testutils.CentralCA, route.Spec.TLS.DestinationCACertificate)
+
+	secret := &v1.Secret{}
+	err = fakeClient.Get(context.TODO(), client.ObjectKey{Name: centralDbSecretName, Namespace: centralNamespace}, secret)
+	require.NoError(t, err)
+	password, ok := secret.Data["password"]
+	require.True(t, ok)
+	assert.True(t, len(password) > 0)
```

Comment:
There are pre-defined existing assertion (asset.NotEmpty could work too):

```suggestion
	assert.Len(t, len(password) > 0)
```

Reply:
`Len` doesn't work in this case (it compares against an exact value), I changed to `NotEmpty`